templates/careers/hiring-process/index.html

```diff
@@ -16,29 +18,30 @@
 
 {% block content %}
 
-  <section class="p-section--hero">
-    <div class="row--50-50">
+  <section class="p-section--hero p-section--shallow">
+    <div class="row--50-50-on-large">
       <div class="col">
         <div class="p-section--shallow">
           <h1>Canonical's hiring process</h1>
         </div>
+        <div class="p-section--shallow">
+          <p>
+            Understand how we approach hiring talent at Canonical, creating a fair and equal playing field for our applicants across the globe.
+          </p>
+        </div>
       </div>
       <div class="col">
-        <p>
-          Understand how we approach hiring talent at Canonical, creating a fair and equal playing field for our applicants across the globe.
-        </p>
-      </div>
-    </div>
-    <div class="u-fixed-width">
-      <div class="p-image-container--cinematic is-cover u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/9acad1b0-department-sales-%402x.jpg",
-                alt="",
-                width="2216",
-                height="834",
-                hi_def=True,
-                loading="auto",
-                attrs={"class": "p-image-container__image"},) | safe
-        }}
+        <div class="u-embedded-media is-cover">
```

Comment:
This modifier only works when using p-image-container so we can remove
```suggestion
        <div class="u-embedded-media">
```